cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -211,6 +226,38 @@ impl Inst {
         Inst::Mov_R_R { is_64, src, dst }
     }
 
+    // TO_ASK: Should be combined with the mov_r_r via some if-statements to handle return?
+    pub(crate) fn sse_scalar_mov_r_r(is_64: bool, src: Reg, dst: Writable<Reg>) -> Inst {
+        debug_assert!(src.get_class() == RegClass::V128);
+        debug_assert!(dst.to_reg().get_class() == RegClass::V128);
+        Inst::SSE_Scalar_Mov_R_R { is_64, src, dst }
+    }
+
+    // TO_ASK: Would think that this should be combined with the mov_r_r via some if-statements?
+    pub(crate) fn return_mov_r_r(is_64: bool, src: Reg, dst: Writable<Reg>) -> Inst {
```

Comment:
Per comments above, I'd prefer that `mov_r_r` continued to be for int-only moves, but renamed to `mov_int_r_r`, and we make a new one that's only for xmm regs, called `mov_xmm_r_r`.